.aws/task-definition-template.json

```diff
@@ -1,31 +1,31 @@
 {
   "containerDefinitions": [
     {
-      "name": "tis-template",
-      "image": "430723991443.dkr.ecr.eu-west-2.amazonaws.com/tis-template:latest",
+      "name": "tis-trainee-credentials",
+      "image": "430723991443.dkr.ecr.eu-west-2.amazonaws.com/tis-trainee-credentials:latest",
       "secrets": [
         {
           "name": "SENTRY_DSN",
-          "valueFrom": "tis-template-sentry-dsn"
+          "valueFrom": "tis-trainee-credentials-sentry-dsn"
         }
       ],
       "logConfiguration": {
         "logDriver": "awslogs",
         "options": {
-          "awslogs-group": "awslogs-${environment}-tis-template",
+          "awslogs-group": "awslogs-${environment}-tis-trainee-credentials",
           "awslogs-region": "eu-west-2",
-          "awslogs-stream-prefix": "awslogs-tis-template"
+          "awslogs-stream-prefix": "awslogs-tis-trainee-credentials"
         }
       },
       "portMappings": [
         {
-          "containerPort": 8080
+          "containerPort": 8209
```

Comment:
8209 is taken by the NDW exporter, I set the infra up to use 8210

Reply:
It is updated to 8210